src/v2/view-builder/views/password/EnrollFactorPasswordView.js

```diff
@@ -41,12 +40,13 @@ export default BaseFactorView.extend({
     return ModelClass.extend({
       local,
       validate: function () {
-        return ValidationUtil.validateFieldsMatch(
-          this,
-          'credential.value',
-          'confirmPassword',
-          loc('password.error.match', 'login'),
-        );
+        if (this.get('credentials.passcode') !== this.get('confirmPassword') &&
```

Comment:
nit:
looks like there is no reason to not reuse `ValidationUtil.validateFieldsMatch`

Reply:
does returning an OR of two calls work how I expect? It just didn't look readible

Reply:
ah, I missed it.